src/h5web/metadata-viewer/EntityInfo.tsx

```diff
@@ -34,20 +57,21 @@ function EntityInfo(props: Props): ReactElement {
             </td>
           </tr>
         )}
-        {isDataset(entity) && (
+        {isMyDataset(entity) && (
           <tr>
             <th scope="row">Shape</th>
             <td>
-              {hasSimpleShape(entity)
+              {hasMySimpleShape(entity)
                 ? renderShapeDims(entity.shape.dims)
                 : entity.shape.class}
             </td>
           </tr>
         )}
+        {isMyLink(entity) && <LinkInfo link={entity.rawLink} />}
```

Comment:
Link metadata are now displayed under the same table section, since links have been promoted to "entities". Note, however, that I still pass the `rawLink` for simplicity (at least for now).